backend/maint-scripts/update_schedule_blobs.py

```diff
@@ -0,0 +1,180 @@
+    ],
+}
+
+
+def move_blob(old_url: str, flag_name: str, kind: str) -> CreateBlobSchema | None:
+    """Fetch blob from old_url, compute checksum, and move to new location"""
+    try:
+        fetch_response = requests.get(
+            old_url,
+            timeout=REQUESTS_TIMEOUT,
+            auth=(BLOB_STORAGE_USERNAME, BLOB_STORAGE_PASSWORD),
+        )
+        fetch_response.raise_for_status()
+        blob_data = fetch_response.content
+    except Exception as exc:
+        logger.warning(f"Failed to fetch {old_url}, exc: {exc!s}")
+        return None
+
+    checksum = hashlib.sha256(blob_data).hexdigest()
+
+    filename = generate_blob_name_uuid(kind)
+    new_private_url = f"{BLOB_PRIVATE_STORAGE_URL}/{filename}"
+    new_public_url = f"{BLOB_PUBLIC_STORAGE_URL}/{filename}"
+
+    headers = {"Destination": new_private_url}
+    try:
+        move_response = requests.request(
+            "MOVE",
+            old_url,
+            headers=headers,
+            timeout=REQUESTS_TIMEOUT,
+            auth=(BLOB_STORAGE_USERNAME, BLOB_STORAGE_PASSWORD),
+        )
+        move_response.raise_for_status()
+    except Exception as exc:
+        logger.warning(f"Failed to move {old_url} to {new_private_url}, exc: {exc!s}")
+        return None
+
+    return CreateBlobSchema(
+        flag_name=flag_name,
+        kind=kind,
+        url=AnyUrl(new_public_url),
+        checksum=checksum,
+    )
+
+
+def main():
+    with Session.begin() as session:
+        for offliner_id, flag_mappings in FLAG_MAPPINGS.items():
+            logger.info(f"Processing offliner: {offliner_id}")
+            offliner = get_offliner(session, offliner_id)
+
+            for schedule in session.execute(
+                select(Schedule)
+                .where(
+                    Schedule.config["offliner"]["offliner_id"].astext == offliner_id,
+                )
+                .options(selectinload(Schedule.offliner_definition))
+            ).scalars():
+                logger.info(f"Processing schedule {schedule.name} blobs...")
+                offliner_definition = create_offliner_definition_schema(
+                    schedule.offliner_definition
+                )
+                schedule_config = ScheduleConfigSchema.model_validate(
+                    {
+                        **schedule.config,
+                        "offliner": create_offliner_instance(
+                            offliner=offliner,
+                            offliner_definition=offliner_definition,
+                            data=schedule.config["offliner"],
+                            skip_validation=True,
+                        ),
+                    },
+                    context={"skip_validation": True},
+                )
+                for flag in flag_mappings:
+                    old_url = getattr(schedule_config.offliner, flag.flag_name)
+                    if not old_url:
+                        logger.warning(
```

Comment:
Why a warning? all flags are optional AFAIK, so most recipes will not have the flag, this is normal